src/projectscene/view/clipsview/clipslistmodel.cpp

```diff
@@ -301,6 +305,26 @@ bool ClipsListModel::changeClipTitle(const ClipKey& key, const QString& newTitle
     return ok;
 }
 
+QVariant ClipsListModel::getSortedNeighbor(const ClipKey& key, int offset) const
```

Comment:
I think now we can just call it `neighbor`, or I think it's better to add two methods - the `next` and the `prev`

Reply:
added next() and prev() methods + left neighbor() as a private member